evalml/tests/preprocessing_tests/test_split_data.py

```diff
@@ -29,6 +30,8 @@ def test_split_data(
         X, y = X_y_regression
     problem_configuration = None
     if is_time_series(problem_type):
+        if is_multiseries(problem_type):
+            pytest.skip("Multiseries time series is tested separately")
```

Comment:
Instead of skipping could we mock out the multi series split and asset that it's called?